subprojects/configuration-cache/src/main/kotlin/org/gradle/configurationcache/ConfigurationCacheState.kt

```diff
@@ -754,6 +755,7 @@ class ConfigurationCacheState(
         BuildOperationFiringSettingsPreparer(
```

Comment:
Can't these synthetic build operations be removed as written in the spec ?

Reply:
Yes, that will be in the next set of changes. I just wanted to do the minimum to add the new events.